utils/include/utils/resource_location.h

```diff
@@ -33,9 +32,12 @@ class resource_location
     std::string get_path() const;
     std::string to_presentable(bool debug = false) const;
 
+    void to_directory();
```

Comment:
why is this needed?

Reply:
The function was removed but the behavior is maintained by calling `join("")` for the discussed reason.